packages/ui-kit/src/hooks/useCounter/CounterMetric.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, RelativeTimeRange, useStorybookAccessToken, quotedStringRegex } from '../../helpers'
+import { useCounter } from './useCounter'
+
+const args = {
+  id: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: { id: process.env.STORYBOOK_METRIC_ID ?? '' },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 30
+    }
+  },
+  name: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: { name: process.env.STORYBOOK_METRIC_UNIQUE_NAME_1 ?? '' },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  },
+  custom: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: {
+      custom: {
+        dataPool: {
+          id: process.env.STORYBOOK_DATAPOOL_ID ?? ''
+        },
+        expression: 'SUM(taco_total_price) / SUM(quantity)'
+      }
+    },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  },
+  sum: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: {
+      sum: {
+        dataPool: {
+          id: process.env.STORYBOOK_DATAPOOL_ID ?? ''
+        },
+        measure: {
+          columnName: 'quantity'
+        }
+      }
+    },
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  }
+}
+
+const meta: Meta = {
+  title: 'Components/Metric',
+  tags: ['pattern'],
+  parameters: {
+    imports: ['useCounter', 'useAccessToken'],
+    isFunction: true,
+    codeTemplate: storybookCodeTemplate,
+    transformBody: (body: string) =>
+      body
+        .replace(
+          'function',
+          `      
+      function`
+        )
+        .replace(`useStorybookAccessToken(axiosInstance)`, 'useAccessToken()')
+        .replace(
+          `{
+      ...args.id,
+      accessToken
+    }`,
+          `${JSON.stringify(args.id)}`
+        )
+        .replace(
+          `{
+      ...args.name,
+      accessToken
+    }`,
+          `${JSON.stringify(args.name)}`
+        )
+        .replace(
+          `{
+      ...args.custom,
+      accessToken
+    }`,
+          `${JSON.stringify(args.custom)}`
+        )
+        .replace(
+          `{
+      ...args.sum,
+      accessToken
+    }`,
+          `${JSON.stringify(args.sum)}`
+        )
+        .replace(process.env.STORYBOOK_METRIC_ID ?? '', '<PROPELL_METRIC_ID>')
+        .replace(process.env.STORYBOOK_DATAPOOL_ID ?? '', '<PROPELL_DATAPOOL_ID>')
```

Comment:
Please find/replace all occurrences of `PROPELL` with `PROPEL`:

```suggestion
        .replace(process.env.STORYBOOK_DATAPOOL_ID ?? '', '<PROPEL_DATAPOOL_ID>')
```